Baeldung/scala-tutorials
# [WIP] Scala fmt config changes

## Verification Steps for this PR:

Note: Moved the 3 modules from `scala-2-modules` back to main directory

- Add a "retest this please" comment in the PR (https://github.com/Baeldung/scala-tutorials/pull/620) and wait for the build to be over. Keep a note of the build number of this build from jenkins.
- Once the build passed, go to this scastie link (https://scastie.scala-lang.org/MYJ0YuRAS4CNmCcM8uWXOQ) and replace the build number line number 3 with the jenkins build number and then run the scastie script. 
- This will print the number of tests executed and passed as part of the PR
- Now, Add a "retest this please" comment for this PR and wait for this to complete. Same way as before find the build number.
- Use the same scastie link : https://scastie.scala-lang.org/MYJ0YuRAS4CNmCcM8uWXOQ and set the jenkins build number and run the scastie script. 
- Check the number of passed tests. This should be almost same(+- 5 normally ) as the previous run of other PR
- Additionally, repeat the same for this PR(https://github.com/Baeldung/scala-tutorials/pull/619). This is a PR created just to make sure that tests are failing. The number of successful tests should be approx 10 less than prev case. Also it should show that the PR doesn't meet merge criteria due to build failure
